templates/Wettbewerbe/eine_veranstaltung.html

```diff
@@ -21,4 +21,28 @@ <h2>Veranstaltung {{ veranstaltung.name }} ({{ veranstaltung.art }})</h2>
       <li><a style="color: gray;" href="mich_eintragen/">[meine Teilnahme eintragen]</a></li>
     {% endif %}
   </ul>
+  {% if veranstaltung.wettbewerbe.all %}
+  <p>Es sind folgende Wettbewerbe bei dieser Veranstaltung ausgetragen worden</p>
+  <ul>
+    {% for w in veranstaltung.wettbewerbe.all %}
+      <li><a href={{ w.get_absolute_url }}>{{ w.name }}</a>
+      {% if w.erfolge.all %}
+      <br />Folgende Erfolge wurden dabei erreicht
+      <ul>
+      {% for e in w.erfolge.all %}
+      <li>
+        {% if e.person %}
+        <a href="{% url 'Wettbewerbe:eine_person' e.person.pk %}">{{ e.person.name }}</a>
+        {% else %}
+          {{ e.nur_name }}
+        {% endif %}
+         - {{ e.name }}
+      </li>
+      {% endfor %}
+      </ul>
+      {% endif %}
```

Comment:
hier drüber stimmt was nicht mit den Einrückungen, sehr schwer lesbar...
man könnte das auch in eine model-Funktion oder einen templatetag packen; falls das noch an anderen Stellen vorkommen soll, hmm...